test/integration_tests/prototype/ctc_decoder_integration_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from torchaudio.prototype.ctc_decoder import lexicon_decoder, download_pretrained_files
+
+
+@pytest.mark.parametrize("model", ["librispeech", "librispeech-3-gram"])
+def test_download_pretrain(model):
+    # smoke test for downloading pretrained files
+    _ = download_pretrained_files(model)
+
+
+@pytest.mark.parametrize("model", ["librispeech", "librispeech-3-gram"])
+def test_decoder_from_pretrained(model):
+    # smoke test for constructing decoder from pretrained files
+    pretrained_files = download_pretrained_files(model)
+
+    _ = lexicon_decoder(
```

Comment:
Since we are constructing the decode object, how about actually decoding something?

I think we can upload some emission data to S3 and fetch it for the sake of testing here, similar to 

[`test_finetune_asr_model`](https://github.com/pytorch/audio/blob/7444f56819f679f68eee8bf915bbb74be3da0e40/test/integration_tests/wav2vec2_pipeline_test.py#L84) but starting from emission instead of waveform.

Reply:
sure, I added a function that generates/returns a random emission if that's sufficient

Reply:
> is there a reason to upload/fetch a sample emission?

That way, we can rest assured that the decode is generating a meaningful result.